company-box.el

```diff
@@ -62,7 +62,7 @@
 
 (require 'subr-x)
 (require 'dash)
-(require 'dash-functional)
+(require 'dash)
```

Comment:
There is already `(require 'dash)` in this list (see PR #170).

More importantly, you can't replace `dash-functional` with `dash` without bumping the required Dash version to at least 2.18.0 (see PR #152).

Reply:
Thanks for the PR, I guess I did not realize `dash` version in here as it is a dependency for multiple packages and it's already updated to latest version. Probably @sebastiencs can find time to merge your PR's soon.